doc/classes/NavigationMeshGenerator.xml

```diff
@@ -12,12 +12,14 @@
 			<argument index="0" name="nav_mesh" type="NavigationMesh" />
 			<argument index="1" name="root_node" type="Node" />
 			<description>
+				Bakes navigation data to the provided [code]nav_mesh[/code] by parsing child nodes under the provided [code]root_node[/code] or a specific group of nodes for potential source geometry. The parse behaviour can be controlled with the [code]geometry/parsed_geometry_type[/code] and [code]geometry/source_geometry_mode[/code] settings on the [NavigationMesh] resource.
```

Comment:
Use links to members:

```suggestion
				Bakes navigation data to the provided [code]nav_mesh[/code] by parsing child nodes under the provided [code]root_node[/code] or a specific group of nodes for potential source geometry. The parse behaviour can be controlled with the [member NavigationMesh.geometry/parsed_geometry_type] and [member NavigationMesh.geometry/source_geometry_mode] properties on the [NavigationMesh] resource.
```

Reply:
Why ... why do I always forget this member thing :(